### ISSUE TITLE
Modpress or move with keyboard unzooms windows

### ISSUE THREAD

**Issue by CRAG666** [jbuchermn/newm#102]
This is bad easy a video is shown. But even if I explain what happens.
For example when I edit videos I like to have a larger kdenlive window, and other smaller windows, the problem is that when I move with the keyboard the zoom is canceled, this also happens with modpress

https://user-images.githubusercontent.com/34254373/163449081-07939bcc-c051-4c5b-a054-df5becac77ba.mp4

**Comment by jbuchermn** [comment]
This is intentional, scaling, as in overview mode, does not look nice with full screen windows. Especially animations have slot of rounding artifacts.

If you don't mind the visual aspects, I'll take a look into adding a config option to disable this behaviour.

**Comment by CRAG666** [comment]
Well really the problem is that the zoom is reset when you press mopress. But this only happens when the windows are not the same size

**Comment by jbuchermn** [comment]
Yup as mentioned above this is intentional, but if it's easily possible we can enable configuring it differently

**Comment by CRAG666** [comment]
@jbuchermn  thanks :)

**Comment by jbuchermn** [comment]
You can now set `view.sticky_fullscreen` to `True` to disable this behaviour.

**Comment by jbuchermn** [comment]
Can you describe a bit better what you mean? Unfortunately your original video is no longer there and it's a bit hard to follow your description.

Switching to fullscreen and then swiping between the windows does no longer exit fullscreen mode, going to overview neither, but 4-finger-swiping to zoom still does - do you mean that?

**Comment by CRAG666** [comment]
https://user-images.githubusercontent.com/34254373/179030664-b56a41d9-89a4-4e9b-b05d-09e91ea8a43b.mp4

**Comment by CRAG666** [comment]
At the end of the video it shows what I do not want to happen, I have 4 window, one of them is larIn normal way, the latter is what I don't want to happen to be kept in mod zoom

**Comment by CRAG666** [comment]
@jbuchermn  In the video I have 4 windows, one of them is larger than the others, so I make mod+u to zoom, I sail for the sale but at the moment I return with mod+j the zoom is canceled, this behavior is the one thatI don't want it to happen, I want it to be kept in zoom mode even if the windows are of different size

**Comment by CRAG666** [comment]
@jbuchermn  I am willing to receive a no with this characteristic.However, it is quite useful with applications such as Kdenlive, since it would work in too much to those who have small monitors or a only monitor (right now I do not have a second monitod).Thank you very much for your excellent job
